src/test/java/arb/logic/commands/client/ListClientCommandTest.java

```diff
@@ -33,6 +33,12 @@ public void execute_listIsNotFiltered_showsSameList() {
                 ListClientCommand.MESSAGE_SUCCESS, expectedModel);
     }
 
+    @Test
+    public void execute_currentListShownProject_success() {
+        assertCommandSuccess(new ListClientCommand(), ListType.PROJECT, ListType.CLIENT, model,
```

Comment:
Shouldn't it be ListType.CLIENT, ListType.PROJECT? Because the 2nd parameter is currentListBeingShown and 3rd parameter is listToBeShown